capability/capability_test.go

```diff
@@ -21,17 +21,31 @@ const (
 	maxLastCap = CAP_CHECKPOINT_RESTORE
 )
 
-func TestLastCap(t *testing.T) {
-	last, err := LastCap()
+// checkErrLinux does two things. First, it fails the test if err is:
+//   - not nil on Linux;
+//   - nil on other platforms.
+//
+// Returns true for Linux, false for other platforms.
+func checkErrLinux(t testing.TB, err error) bool {
 	switch runtime.GOOS {
 	case "linux":
 		if err != nil {
+			t.Helper()
```

Comment:
I have caught this patter from go sources. In many places they do it this way (supposedly because `t.Helper()` is costly, and if we can avoid it we should -- but I haven't checked for sure, just thought that they know what they are doing 😁

Reply:
This code is removed, and this patch is now moved to #162.